chain/blocksync/blocksync.go

```diff
@@ -55,8 +48,8 @@ func ParseBSOptions(optfield uint64) *BSOptions {
 }
 
 const (
-	BSOptBlocks   = 1 << 0
-	BSOptMessages = 1 << 1
+	BSOptBlocks = 1 << iota
```

Comment:
this breaks the protocol, no?

Reply:
It doesn't, it is just a nicer way to write this. The values should be the same.

Reply:
Confirmed, values are the same.

Reply:
Confirmed, values are the same.